components/enrichers/enricher_test.go

```diff
@@ -0,0 +1,107 @@
+package enrichers
```

Comment:
can u pls add tests for the other public methods as well? Imo these base-components should have higher coverage than the rest, just because issues cascade to all enrichers